<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Preparation Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnSort">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Sort the currently selected column (A-Z or 0-9)</string>
        </property>
        <property name="text">
         <string>Sort</string>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnChangeType">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Change Data Type</string>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRename">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string extracomment="Sets the column with text to be searched. Select a column below and click this button to set it."/>
        </property>
        <property name="text">
         <string>Rename selected column</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDropCol">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string extracomment="Tags all rows with a single column (Keyword). All keywords will appear here seperated by a comma."/>
        </property>
        <property name="text">
         <string>Delete selected column</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::DoubleClicked</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblUpdate">
      <property name="text">
       <string/>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="act_open"/>
    <addaction name="separator"/>
    <addaction name="act_quit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="act_about"/>
   </widget>
   <addaction name="menuOpen"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="menu_openFile"/>
   <addaction name="separator"/>
   <addaction name="menu_save"/>
   <addaction name="menu_undo"/>
   <addaction name="menu_redo"/>
   <addaction name="separator"/>
   <addaction name="menu_trim"/>
   <addaction name="menu_nullVals"/>
   <addaction name="separator"/>
   <addaction name="menu_desc"/>
   <addaction name="menu_info"/>
   <addaction name="separator"/>
   <addaction name="menu_help"/>
  </widget>
  <action name="act_open">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="act_quit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="act_trim">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Trim Columns</string>
   </property>
   <property name="toolTip">
    <string>Trim Column Data</string>
   </property>
  </action>
  <action name="act_clearTable">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Clear Table</string>
   </property>
  </action>
  <action name="act_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="menu_openFile">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="menu_trim">
   <property name="text">
    <string>Trim Column</string>
   </property>
   <property name="toolTip">
    <string>Trim Column Data</string>
   </property>
  </action>
  <action name="menu_nullVals">
   <property name="text">
    <string>Set Null Values</string>
   </property>
   <property name="toolTip">
    <string>Set Null Values</string>
   </property>
  </action>
  <action name="actionClear_Null_Values">
   <property name="text">
    <string>Set null values</string>
   </property>
  </action>
  <action name="actionTrim_digits">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Trim digits</string>
   </property>
  </action>
  <action name="menu_save">
   <property name="text">
    <string>Save File</string>
   </property>
   <property name="toolTip">
    <string>Save File</string>
   </property>
  </action>
  <action name="menu_undo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo last changes</string>
   </property>
  </action>
  <action name="menu_redo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo last changes</string>
   </property>
  </action>
  <action name="menu_desc">
   <property name="text">
    <string>Describe</string>
   </property>
   <property name="toolTip">
    <string>Describe dataset</string>
   </property>
  </action>
  <action name="menu_info">
   <property name="text">
    <string>Info</string>
   </property>
   <property name="toolTip">
    <string>Info</string>
   </property>
  </action>
  <action name="menu_help">
   <property name="text">
    <string>Help</string>
   </property>
   <property name="toolTip">
    <string>Open the user guide on GitHub in the default browser</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
